bundle/Resources/public/js/views/plugins/cof-createcontent-universaldiscoveryplugin.js

```diff
@@ -50,6 +52,30 @@ YUI.add('cof-createcontent-universaldiscoveryplugin', function (Y) {
         },
 
         /**
+         * Checks if `browse` method is present in the methods definitions.
+         * If not, then sets the visible method to `finder`
+         *
+         * @method _checkBrowseMethodPresence
+         * @protected
+         * @param event {Object} event facade
+         */
+        _checkBrowseMethodPresence: function (event) {
+            var browseMethod;
+
+            if (!event.newVal || !this.get('forceVisibleMethod')) {
+                return;
+            }
+
+            browseMethod = this.get('methods').filter(function (method) {
+                return method.get('id') === 'browse';
+            })[0];
+
+            if (!browseMethod) {
+                this.set('visibleMethod', 'finder');
```

Comment:
one method should be always set. If `browse` view exists the `browse` method should be set. 
And also I think we should reset the `forceVisibleMethod`, I don't know if this flag will stay when the UDW is opened again.